server/src/test/java/org/elasticsearch/search/aggregations/metrics/TDigestPercentilesAggregatorTests.java

```diff
@@ -100,14 +100,14 @@ public void testSomeMatchesNumericDocValues() throws IOException {
             assertEquals(tdigest.state.centroidCount(), 7L);
             assertEquals(8.0d, tdigest.percentile(100), 0.0d);
             assertEquals("8.0", tdigest.percentileAsString(100));
-            assertEquals(6.98d, tdigest.percentile(88), 0.0d);
-            assertEquals("6.98", tdigest.percentileAsString(88));
-            assertEquals(1.0d, tdigest.percentile(33), 0.0d);
-            assertEquals("1.0", tdigest.percentileAsString(33));
-            assertEquals(1.0d, tdigest.percentile(25), 0.0d);
-            assertEquals("1.0", tdigest.percentileAsString(25));
-            assertEquals(0.0d, tdigest.percentile(1), 0.0d);
-            assertEquals("0.0", tdigest.percentileAsString(1));
+            assertEquals(4.0d, tdigest.percentile(75), 0.01d);
```

Comment:
Why not keep checking 88, 33, 35 and 1 percentiles?

Reply:
I think these were arbitrarily selected to produce round values.